internal/synclet/client_adapter.go

```diff
@@ -28,6 +33,26 @@ func NewGRPCClient(conn *grpc.ClientConn) *SyncletCli {
 	return &SyncletCli{del: proto.NewSyncletClient(conn), conn: conn}
 }
 
+func protoLogLevelToLevel(protoLevel proto.LogLevel) (logger.Level, error) {
+	var level logger.Level
+	switch protoLevel {
+	case proto.LogLevel_INFO:
+		level = logger.InfoLvl
+	case proto.LogLevel_VERBOSE:
+		level = logger.VerboseLvl
+	case proto.LogLevel_DEBUG:
+		level = logger.DebugLvl
+	default:
+		return logger.InfoLvl, fmt.Errorf("unknown log level '%v'", protoLevel)
```

Comment:
hmmm...not sure this is a good idea, because it makes the protocol non-backwards compatible